big_tests/tests/inbox_SUITE.erl

```diff
@@ -440,6 +458,35 @@ reset_unread_counter(Config) ->
     check_inbox(Mike, [#conv{unread = 0, from = Kate, to = Mike, content = <<"Hi mike">>}])
                                                 end).
 
+reset_unread_counter_and_show_only_unread(Config) ->
+  escalus:story(Config, [{kate, 1}, {mike, 1}, {alice, 1}], fun(Kate, Mike, Alice) ->
+    % Kate sends message to Mike
+    MsgId =  <<"123123">>,
+    Msg1 = escalus_stanza:set_id(escalus_stanza:chat_to(Mike, <<"Hi mike">>), MsgId),
+    escalus:send(Kate, Msg1),
+    M1 = escalus:wait_for_stanza(Mike),
+    escalus:assert(is_chat_message, M1),
+    %% Mike has one unread message
```

Comment:
rather "has one CONVERSATION with UNREAD messages".
Generally I am not a fun of comments, the code speaks for itself most of the time (this time as well) and comments tend to lie :o

Reply:
So should I change the comment or just remove it? :)

Reply:
IMHO remove it :)